Dockerfile

```diff
@@ -44,3 +45,7 @@ RUN go install github.com/go-delve/delve/cmd/dlv@v1.8.0
 
 # overwrite server and include debugger
 COPY --from=builder /opt/app-root/src/rekor-server_debug /usr/local/bin/rekor-server
+
+FROM deploy as test
+# overwrite server with test build with code coverage
```

Comment:
this is just an information question: does this mean that this happens all the time, including for prod builds?

Reply:
The test coverage build always runs as part of the builder step, but is only copied as the final binary if you're targeting "test". So comparing docker-compose.test.yaml to docker-compose.yaml, the test compose file targets "test" whereas the prod file targets "deploy" https://github.com/sigstore/rekor/blob/7652a619d8389632b01e489f6ebd076d46cdbded/docker-compose.yml#L83

Reply:
AH THANK YOU!